asset_dashboard/static/js/PortfolioPlanner.js

```diff
@@ -589,7 +595,10 @@ class PortfolioPlanner extends React.Component {
         'category': project.category,
         'project_manager': project.project_manager,
         'phase': project.phase,
-        'status': project.status,
+        'phase_status': project.phase_status,
+        'actual_cost': project.actual_cost,
+        'project_status': project.project_status,
```

Comment:
Just an fyi that the terminology used in the planner is a little confusing.

It looks like each row describes a separate project or maybe every project's phase, but each row is actually every funding stream, connected to a phase, that belongs to a project. So there's a little bit of a disconnect between the variable names in the view vs. what's in here.

I renamed this field to help with clarity just a bit. I'm sure there'll be other spots we come back to for clarity's sake.